cores/esp8266/WString.h

```diff
@@ -68,7 +71,9 @@ class String {
         explicit String(unsigned long, unsigned char base = 10);
         explicit String(float, unsigned char decimalPlaces = 2);
         explicit String(double, unsigned char decimalPlaces = 2);
-        ~String(void);
+        inline ~String() {
```

Comment:
Just curious, but what extra does `inline` add to a function implemented in a .h file?
I always thought implementing a function in a .h file always implies `inline`.

Reply:
as you said, function bodies inside a class definition (typically in .h) are implicitly marked as `inline`.

like extra arithmetic parentheses, may be removed later.